### PR TITLE
EWC-199 stubs for new fog controller routes

### PR DESCRIPTION
**Pull request by Railag**

### CODE REVIEW

File: src/routes/controller.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                .status(200)
+                .send({
+                    "status": "ok",
+                    "timestamp": Date.now(),
+                })
+        }
+    },
+    {
+        method: 'get',
+        path: '/api/v3/emailActivation',
```
Comment (baghbidi) [inline src/routes/controller.js]: change it to `/api/v3/email-activation` and update swagger

File: src/routes/controller.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        middleware: (req, res) => {
+            res
+                .status(200)
+                .send(req.body)
+        }
+    },
+    {
+        method: 'get',
+        path: '/api/v3/fogTypes',
```
Comment (baghbidi) [inline src/routes/controller.js]: change it to `/api/v3/fog-types` and update swagger

File: src/routes/diagnostics.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+module.exports = [
+    {
+        method: 'post',
+        path: '/api/v3/iofog/microservices/:id/imageSnapshot',
```
Comment (baghbidi) [inline src/routes/diagnostics.js]: change it to `/api/v3/iofog/microservices/:id/image-snapshot` and update swagger

File: src/routes/diagnostics.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        middleware: (req, res) => {
+            res
+                .status(200)
+                .send(req.body)
+        }
+    },
+    {
+        method: 'get',
+        path: '/api/v3/iofog/microservices/:id/imageSnapshot',
```
Comment (baghbidi) [inline src/routes/diagnostics.js]: same here

File: src/routes/user.js [hunk 5]
```diff
@@ -1,11 +1,92 @@
...
+        path: '/api/v3/user/signup',
+        middleware: (req, res) => {
+            res
+                .status(200)
+                .send(req.body)
+        }
+    },
+    {
+        method: 'get',
+        path: '/api/v3/user/signup/resendActivation',
```
Comment (baghbidi) [inline src/routes/user.js]: `/api/v3/user/signup/resend-activation`

**Comment by Railag** [comment]
Fixed endpoints, updated indents & created pull request with swagger naming fixes

**Pull request merged**